using-timescaledb/compression.md

```diff
@@ -295,14 +295,81 @@ SELECT compress_chunk(i) from show_chunks('conditions', newer_than, older_than)
 
 ### Decompressing Chunks [](decompress-chunks)
 
-Next we will walk through what to do in the event that you need to backfill or
-update data that lives in a compressed chunk.
+In order to decompress individual chunks, you can run a `decompress_chunk`
+command in much the same way you can manually compress an individual chunk:
 
-TimescaleDB does not support inserts or updates into a compressed chunk. To insert
-or update data, we must first decompress the target chunks. But before we can do
-that, we need to turn off our compression policy. Otherwise that policy will attempt
-to re-compress the chunks that we are currently working on (not the desired result).
-To accomplish this we first find the job_id of the policy using:
+``` sql
+SELECT decompress_chunk('_timescaledb_internal._hyper_2_2_chunk');
+```
+
+Similar to above, you can also decompress a set of chunks based on a
+time range by first looking up this set of chunks via `show_chunks`:
+
+``` sql
+SELECT decompress_chunk(i) from show_chunks('conditions', newer_than, older_than) i;
+```
```

Comment:
There is an additional way to get chunks matching constraints which is especially useful if you have more complex constraints (e.g. space partitioning):

```
dev=# select tableoid::regclass FROM metrics WHERE time = '2000-01-01' AND device_id = 1 GROUP BY tableoid;
                 tableoid
------------------------------------------
 _timescaledb_internal._hyper_72_37_chunk
(1 row)
```